Xamarin.Forms.Core/Binding.cs

```diff
@@ -139,7 +199,7 @@ internal override BindingBase Clone()
 				Source = Source,
 				UpdateSourceEventName = UpdateSourceEventName,
 				TargetNullValue = TargetNullValue,
-				FallbackValue = FallbackValue,
+				FallbackValue = FallbackValue
```

Comment:
nope, that was correct :) and on purpose

Reply:
Oops inadvertent :)